### DISCUSSION TITLE
How can I locate products uploaded via "JSON_LISTINGS_FEED" FeedType in Seller Central?

### DISCUSSION THREAD

**Discussion by bstmartinswe** [amzn/selling-partner-api-models#3846] [General]
I used the Create Feed API with "JSON_LISTINGS_FEED" to upload product data. Although I can find the product using the getListingsItem API with SKU, I cannot locate it in Seller Central - Inventory.

Thanks!

**Comment by vaidas-lungis** [comment]
Check fix your product page https://sellercentral.amazon.com/fixyourproducts

https://www.ecomengine.com/sell-on-amazon/fix-your-products-amazon
